config/index.ts

```diff
@@ -52,9 +53,8 @@ const config: IConfig = {
       environment: process.env.ROLLBAR_ENV
     },
     session: {
-      // eslint-disable-next-line @typescript-eslint/camelcase
-      header_name: 'authorization',
-      secret: process.env.NODE_API_SESSION_SECRET
+      header_name: 'authorization', // eslint-disable-line @typescript-eslint/camelcase
```

Comment:
This rule should be for variable names only, prop objects are normally snake_case in API in/out